src/recipes/mulled-wine.json

```
@@ -0,0 +1,38 @@
+{
+    "name": "<Mulled Wine>",
```

```suggestion
    "name": "Mulled Wine",
```
You can remove all the less-than/greater-than symbols out. They are just placeholders.

Thanks!

---

All edits have been made - thanks very much.

---

Please let me know if I need to do anything further on my end.  Thanks!!